plugins/crd/controller/nodeconfig/node_config_controller.go

```diff
@@ -120,21 +123,27 @@ func (c *Controller) Init() error {
 			newEvent.resource = new
```

Comment:
Let's change it to oldObj, newObj so it won't be confused with keyword "new"

Reply:
agree